DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -2081,11 +2081,11 @@ public async Task<IReadOnlyList<DiscordAuditLogEntry>> GetAuditLogsAsync(int? li
                                     };
                                     break;
 
-                                case "application_id": //No idea what this is or does and can't test it so /shrug I guess this probably works
+                                case "application_id": //Why the fuck does discord send this as a string if it's supposed to be a snowflake
```

Comment:
Javascript.

Reply:
Snowflakes are sent as strings due to "language restraints"

Reply:
Brain restraints